packages/backend/package.json

```diff
@@ -38,6 +38,7 @@
     "@backstage/plugin-todo-backend": "^0.3.12",
     "@frontside/backstage-plugin-graphql-backend": "^0.1.2",
     "@frontside/backstage-plugin-graphql-backend-module-catalog": "^0.2.0",
+    "@internal/backstage-plugin-scaffolder-backend-module-github-pages": "^0.1.0",
```

Comment:
is it not possible to use workspace dependencies in here?

Reply:
Ah, yes, looks like I can 👍